lib/Remotery.h

```diff
@@ -587,19 +608,6 @@ typedef enum
 /* --- Private Details ---------------------------------------------------------------------------------------------------------*/
 
 
-// All possible property types that can be recorded and sent to the viewer
-typedef enum
-{
-    RMT_PropertyType_rmtGroup,
-    RMT_PropertyType_rmtBool,
-    RMT_PropertyType_rmtS32,
-    RMT_PropertyType_rmtU32,
-    RMT_PropertyType_rmtF32,
-    RMT_PropertyType_rmtS64,
-    RMT_PropertyType_rmtU64,
-    RMT_PropertyType_rmtF64,
-} rmtPropertyType;
-
 // A property value as a union of all its possible types
 typedef union rmtPropertyValue
```

Comment:
Q: SHould we make this public as well, or add multiple getter functions to the property api:
`rmt_PropertyGetS32()` vs `rmt_PropertyGetValue()`

Reply:
I am happy to make `rmtProperty` and `rmtPropertyValue` public and maintain their API.

It keeps everything super simple.

I've been currently pondering on the idea of making `Sample` public, as well. Not sure about that one yet...